src/windowsdesktop/src/sfx/Directory.Build.targets

```diff
@@ -4,14 +4,20 @@
 
   <ItemGroup>
     <KnownFrameworkReference
-      Update="Microsoft.NETCore.App"
-      DefaultRuntimeFrameworkVersion="$(MicrosoftNETCoreAppRefVersion)"
-      LatestRuntimeFrameworkVersion="$(MicrosoftNETCoreAppRefVersion)"
-      TargetingPackVersion="$(MicrosoftNETCoreAppRefVersion)" />
+      Update="Microsoft.NETCore.App">
+      <DefaultRuntimeFrameworkVersion Condition="'%(TargetFramework)' == '$(TargetFramework)'">$(MicrosoftNETCoreAppRefVersion)</DefaultRuntimeFrameworkVersion>
+      <LatestRuntimeFrameworkVersion Condition="'%(TargetFramework)' == '$(TargetFramework)'">$(MicrosoftNETCoreAppRefVersion)</LatestRuntimeFrameworkVersion>
+      <TargetingPackVersion Condition="'%(TargetFramework)' == '$(TargetFramework)'">$(MicrosoftNETCoreAppRefVersion)</TargetingPackVersion>
+    </KnownFrameworkReference>
+    <KnownCrossgen2Pack
+      Update="Microsoft.NETCore.App.Crossgen2">
+      <Crossgen2PackVersion Condition="'%(TargetFramework)' == '$(TargetFramework)'">$(MicrosoftNETCoreAppRefVersion)</Crossgen2PackVersion>
```

Comment:
This looks a bit weird. Why not condition this on `NetCurrent`? These version properties should only be applied to the latest TFM anyway.